.travis.yml

```diff
@@ -8,6 +8,7 @@ julia:
     - 1.1
     - 1.2
     - 1.3
+    - 1.4
```

Comment:
Maybe we should just do:

```yml
- 1.0
- 1
- nightly
```

Reply:
That sounds good, also it would hopefully lead to less errors due to api limit in cases someone pushes 3 commits in a row :D .
https://www.google.com/imgres?imgurl=https%3A%2F%2Fimages-na.ssl-images-amazon.com%2Fimages%2FI%2F71eCUzklxnL._SY445_.jpg&imgrefurl=https%3A%2F%2Fwww.amazon.com%2FMeeseeks-Riveter-Cartoon-Comedy-POSTER%2Fdp%2FB0794466WP&tbnid=I0ueMwL5yyB1oM&vet=12ahUKEwjumvjUvdfpAhUGghoKHaM3DOQQMygAegUIARDvAQ..i&docid=qBcU9nhdfXG4zM&w=305&h=445&q=can%20do%20meeseeks&ved=2ahUKEwjumvjUvdfpAhUGghoKHaM3DOQQMygAegUIARDvAQ

Reply:
While we're at it, probably best to add Windows to Travis and remove AppVeyor...

Reply:
And that means just deleting `appveyor.yml`?

Reply:
Delete that file, and add `windows` to the `os` list in `.travis.yml`.

Reply:
CI shuffling could be done in a separate PR; doesn't have to happen here.

Reply:
Ok, I'll make separate PR for ot once this is merged.